src/CorpusLoaders.jl

```diff
@@ -5,19 +5,24 @@ using MultiResolutionIterators
 using InternedStrings
 using Glob
 using StringEncodings
+using CSV
+using DelimitedFiles
```

Comment:
I think this can be delete since using CSV?

Reply:
I am using DelimitedFiles in `StanfordSentimentTreebank.jl`